<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTWizard</class>
 <widget class="QWizard" name="CTWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="options">
   <set>QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="CTWizardPage" name="CTWizardPageInput">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string>Please fill in the following data.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <item row="1" column="1">
       <widget class="QDateEdit" name="qde_execution">
        <property name="date">
         <date>
          <year>2012</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="qle_description"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Day of execution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Order of execution</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="qsb_execution">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="CTWizardPage" name="CTWizardPageOutM">
   <property name="title">
    <string/>
   </property>
   <property name="subTitle">
    <string>Check at least one outcome measure and then click Finish to save the defined scenario into the database.</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Outcome measures:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="qchb_grasps_number">
         <property name="text">
          <string>How many grasps has the infant done?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_grasp_latency">
         <property name="text">
          <string>How much time has the infant spent to grasp the toy?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_grasp_strength">
         <property name="text">
          <string>With which strength has the infant grasped?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_button_presses">
         <property name="text">
          <string>How many times has the infant pressed the button?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_button_latency">
         <property name="text">
          <string>How much time has the infant spent to press the button?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_turn_latency">
         <property name="text">
          <string>How much time did the infant spend to move towards the active wall?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_lean_forward">
         <property name="text">
          <string>Did the infant lean forward with his trunk?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qchb_weight_distribution">
         <property name="text">
          <string>How was the distribution of the weight on the  mat?</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTWizardPage</class>
   <extends>QWizardPage</extends>
   <header>Scenario_Designer/ct_wizardpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
